dashboard/src/components/AppUpgrade/AppUpgrade.test.tsx

```diff
@@ -75,12 +76,40 @@ it("renders an error if it exists", () => {
       repo={repo}
     />,
   );
-  expect(wrapper.find(DeploymentErrors).exists()).toBe(true);
+  expect(wrapper.find(ErrorSelector).exists()).toBe(true);
   expect(wrapper.find(SelectRepoForm).exists()).toBe(false);
   expect(wrapper.find(UpgradeForm).exists()).toBe(false);
+  expect(wrapper.html()).toContain("Sorry! Something went wrong.");
   expect(wrapper).toMatchSnapshot();
 });
 
+it("renders a forbidden error", () => {
```

Comment:
Maybe you can split this to context + case name?

context("when we receive a ForbiddenError error prop")...
it("renders the error message")

or something like that.